mars/oscar/backends/mars/tests/test_pool.py

```diff
@@ -227,6 +227,21 @@ async def test_sub_actor_pool(notify_main_pool):
         await pool.stop()
 
 
+@pytest.mark.asyncio
+async def test_fail_when_create_subpool():
+    config = ActorPoolConfig()
+    my_label = 'computation'
+    main_address = f'127.0.0.1:{get_next_port()}'
+    port = get_next_port()
+    _add_pool_conf(config, 0, 'main', 'unixsocket:///0', main_address)
+    _add_pool_conf(config, 1, my_label, 'unixsocket:///1', f'127.0.0.1:{port}',
+                   env={'my_env': '1'})
```

Comment:
Better to add a comment about what kind of error will be raised when creating the pool.